topics/admin/tutorials/connect-to-compute-cluster/slides.html

```diff
@@ -59,7 +59,7 @@
 ---
 # Exercise
 
-[Running Galaxy jobs with Slurm](./tutorial.html)
+[Running Galaxy jobs with Condor](./tutorial.html)
```

Comment:
It seems the exercise is still using Slurm, no?

```suggestion
[Running Galaxy jobs with Slurm](./tutorial.html)
```